extras/cronwrapper

```diff
@@ -14,11 +14,11 @@
 # LOGGING if true, logs all output to syslog daemon facility
 #         (typically /var/log/daemon.log or /var/log/syslog)
 #
-# Set CONFIGURED to true once you've setup the previous three
-# options.
+# NOTIFYONRETURN is used to define which return codes from plexupdate
```

Comment:
I agree, but it got very bulk when I started to do more. And since I was controlling the pattern, I felt it to be safe. Worst case, it wouldn't match :) Anyway, sleep easy, the scrappy patch is gone ;)